gnosis/cowswap
# Do not allow to wrap for unsupported wallets

# Summary
Closes #1845
Do not allow to wrap ETH or xDAI for unsupported wallets.

# To Test

* Connect to a wallet that is not supported
* Select XDAI/ETH in the From section
* Select a token to buy
* Observe that the wrapping message is not visible any more
